src/main/java/seedu/address/model/appointment/NonConflictingAppointmentList.java

```diff
@@ -47,6 +55,14 @@ public boolean hasConflict(Appointment toCheck) {
         return internalList.stream().anyMatch(toCheck::hasConflict);
     }
 
+    /**
+     * Returns true if the list contains appointments that are in conflict with {@code toCheck}
+     */
+    public boolean hasEditConflict(Appointment toCheck) {
```

Comment:
I don't think this is needed as it functions the same as `hasConflict`.

Reply:
Thank you for pointing out!
Can i keep it for now? because it separates from the functions used by add-appt. 
In the future if there will be some changes, i can change it easily without breaking the code for add-appt.
In the finalised version, if they are the same, i will rename them. 
It has the flow of : hasEditConflictingAppointment -> hasEditAppointment -> editContains 
Otherwise the flow would be: hasConflictingAppointment -> hasAppointment -> editContains.

Reply:
you are right. I removed it